config/build.sh

```diff
@@ -12,8 +12,8 @@ set -e -x
 VERSION="$(date -u +%Y-%m-%d)-$(git rev-parse --short HEAD)"
 
 cd "${BASEDIR}"
-docker build -f cmds/http-gateway/Dockerfile  . -t gcr.io/$CLOUD_PROJECT/http-gateway:$VERSION
-docker build -f cmds/grpc-backend/Dockerfile  . -t gcr.io/$CLOUD_PROJECT/grpc-backend:$VERSION
+docker build -f cmds/http-gateway/Dockerfile  . -t $DOCKER_URL:$VERSION
```

Comment:
Shouldn't there be an "http-gateway" suffix here?  And "grpc-backend" suffix on the next line?